src/app/content/components/Topbar/styled.tsx

```diff
@@ -357,7 +357,10 @@ export const TextResizerDropdown = styled(FilterDropdown)`
     }
   }
 
-  ${theme.breakpoints.mobileMedium(css`
+  ${(props: {
+    mobileVariant: boolean,
+    mobileToolbarOpen: boolean
+  }) => props.mobileVariant !== false && theme.breakpoints.mobileMedium(css`
     margin-left: 0;
```

Comment:
What's this about?

Reply:
There are mobile styles that get applied because of the main toolbar collapses a bit on mobile: 
![image](https://user-images.githubusercontent.com/34174/193419496-4c3b2ad4-5e33-4d80-b403-9df77b7fba47.png)
That ends up looking broken on this new toolbar because it doesn't collapse... maybe we want that? I don't see any mobile designs but I can make the styles more flexible.